sandy/core/endf6.py

```diff
@@ -170,6 +183,22 @@ def get_endf6_file(library, kind, zam, to_file=False):
     >>> tape = sandy.get_endf6_file("jeff_33", 'decay', [380900, 551370, 541350])
     >>> assert type(tape) is sandy.Endf6
 
+    Thermal Neutron Scattering Data from ENDF/B-VII.1. ('all' is tested also)
+    >>> tape = sandy.get_endf6_file("endfb_71", 'tsl', [1, 2, 3])
+    >>> assert type(tape) is sandy.Endf6
+
+    Thermal Neutron Scattering Data from ENDF/B-VIII.0. ('all' is tested also)
+    >>> tape = sandy.get_endf6_file("endfb_80", 'tsl', [1, 2, 3])
+    >>> assert type(tape) is sandy.Endf6
+
+    Thermal Neutron Scattering Data from JEFF-3.3. ('all' is tested also)
+    >>> tape = sandy.get_endf6_file("jeff_33", 'tsl', [1, 2, 3])
+    >>> assert type(tape) is sandy.Endf6
+
+    Thermal Neutron Scattering Data from JENDL-4.0u ('all' is tested also)
```

Comment:
To say that it is tested doesn't mean it is. Just create a notebook for 1 library and upload it on the sandy notebook environment.

Reply:
i dont implement the reading of tsl, only the libraries and  the get_endf6_file method

Reply:
Sure, for that you would need to update `read_section`